submissions/TaniaBondarenko/popup2_task/style.css

```diff
@@ -0,0 +1,101 @@
+/*Common styles*/
```

Comment:
Remove these comments too. It does not need to use it here. It's clear to read for other developers